src/components/PlayerCardContainer/CustomInput.js

```diff
@@ -0,0 +1,13 @@
+import React, {useState} from "react";
+
+import './CustomInput.css'
+
+function CustomInput(props) {
+        return (
+            <input type={props.type} className={props.classes} placeholder={props.placeHolder}
+                   value={props.name? props.name : props.balance} onChange={props.changeHandler}
```

Comment:
Another little piece where I think CustomInput knows too much.  It shouldn't have to differentiate between props.name or props.balance, if you use the same name for whatever you're passing in as the value to be displayed then your component can be dumber, and therefor easier to maintain.